cli/src/formatter.rs

```diff
@@ -389,9 +389,36 @@ fn color_for_name(color_name: &str) -> Result<Color, config::ConfigError> {
         "bright magenta" => Ok(Color::Magenta),
         "bright cyan" => Ok(Color::Cyan),
         "bright white" => Ok(Color::White),
-        _ => Err(config::ConfigError::Message(format!(
-            "invalid color: {color_name}"
-        ))),
+        _ => {
+            if name_or_hex.len() == 7 && name_or_hex.starts_with('#') {
+                color_for_hex(name_or_hex)
+            } else {
+                Err(config::ConfigError::Message(format!(
+                    "invalid color: {}",
+                    name_or_hex
+                )))
+            }
+        }
+    }
+}
+
+fn color_for_hex(color: &str) -> Result<Color, config::ConfigError> {
+    if color.len() == 7 && color.starts_with('#') {
+        let r = u8::from_str_radix(&color[1..3], 16);
+        let g = u8::from_str_radix(&color[3..5], 16);
+        let b = u8::from_str_radix(&color[5..7], 16);
+        match (r, g, b) {
```

Comment:
nit: `from_str_radix()` can parse `+` sign, so it might be better to first check `color.contains(|c: char| !c.is_ascii_hexdigit())`.

https://doc.rust-lang.org/std/primitive.u8.html#method.from_str_radix

Reply:
I did not know that!